src/clojure_tictactoe/game/game_rules.clj

```diff
@@ -27,8 +26,7 @@
     top-left-to-bottom-right-diagonal top-left-corner
     bottom-left-to-top-right-diagonal top-right-corner)))
 
-(defn game-over?
-  [board]
+(defn game-over? [board]
   (not (not (or
```

Comment:
I know you talked with Byron about this on the last PR, but I'm still not entirely clear on why this can't be simplified. Can `board/board-full? board` or `assess-win board` be made to always return booleans?

Reply:
I reworked `assess-win` in order to address this. It no longer exists and has been replaced with `assess-winner` and `winner`. `board-full?` always returned booleans, but both of them had to always return booleans in order to ensure the not-not was unnecessary.